### ISSUE TITLE
FOTT bug report: Missing Labels 

### ISSUE THREAD

**Issue by tianxiangs** [microsoft/OCR-Form-Tools#349]
**Describe the bug**
Some labels are missing in the project

**Screenshots**
![analyze_miss_label](https://user-images.githubusercontent.com/60452964/85807840-f158bc80-b707-11ea-8334-7c8869a273ae.jpg)
![model_miss_aria_label](https://user-images.githubusercontent.com/60452964/85807853-f87fca80-b707-11ea-8d27-18a0eb89610c.png)
![model_name_field_miss_label](https://user-images.githubusercontent.com/60452964/85807857-fb7abb00-b707-11ea-80a2-1f21f24dc900.png)

**Comment by alex-krasn** [comment]
There is additional missing aria-label on scroll
![image](https://user-images.githubusercontent.com/64093224/86517134-559c1000-bddb-11ea-8550-5954762cd136.png)
